src/iotile/models/properties.ts

```diff
@@ -0,0 +1,8 @@
+export class Properties {
+
+  constructor(datas) {
+    for (let data of datas) {
+      this[data.name] = data.value;
```

Comment:
But if you move to using `Property`, then you don't really need get/set functions. The suggestion was only for `Properties`